content/authorization.md

```diff
@@ -1161,11 +1135,6 @@ preceding documentation.
   soon as possible via
   [Cerner Central System Account Management][SYSTEM-ACCOUNTS].  
 
-- Is offline_access supported for healthcare providers?
-
-> Cerner does not currently have support for offline_access
-  for providers.  It is currently under consideration.
```

Comment:
I think this probably still needs to stay in here, unfortunately. We're still a bit out from supporting offline_access for providers.

Reply:
Fixed.